multiqc/templates/default/content.html

```diff
@@ -13,7 +13,12 @@
     {% for s in m.sections %}
       {% if loop.first %}
       <div class="mqc-module-section-first">
-        <h2 id="{{ m.anchor }}">{{ m.name }}</h2>
+        <div class="mqc-module-title-section">
+          <div><h2 class="mqc-module-title" id="{{ m.anchor }}">{{ m.name }}</h2></div>
```

Comment:
I don't think that we need a wrapper `div` here.
```suggestion
          <h2 class="mqc-module-title" id="{{ m.anchor }}">{{ m.name }}</h2>
```